### PR TITLE
Consolidate to single .env

### LINKED ISSUE #701: consolidate env
Merge 2 envs to one.

### PR DESCRIPTION
**Pull request by gvn**
Closes #701

### CODE REVIEW

**Comment by gvn** [comment]
I'm guessing this might require some deployment tweaking?

**Comment by cadecairos** [comment]
nope, this shouldn't affect the deployment - environments on Heroku are applied to the container automatically, not via FS

File: env.default [hunk 1]
```diff
@@ -1,6 +1,41 @@
...
+ALLOWED_HOSTS=localhost
+ASSET_DOMAIN=network.mofoprod.net
+BUILD_TRIGGER_URL=
+BUILD_DEBOUNCE_SECONDS=300
+BUILD_THROTTLE_SECONDS=900
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY=
+SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET=
+SOCIAL_AUTH_LOGIN_REDIRECT_URL=http://localhost:5000/soc/complete/google-oauth2/
+
+# CSP config
```
Comment (cadecairos) [inline env.default]: does our env loader understand comments, since this isn't a bash-style `source` call?

Comment (cadecairos) [reply env.default]: [It reportedly does](https://github.com/joke2k/django-environ)

Comment (gvn) [reply env.default]: It seems to. On the build system side it converts to JSON and just ignores comments as expected.

**Pull request merged**